cloud/kubernetes/helm/values.yaml

```diff
@@ -0,0 +1,239 @@
+# Default values.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+#Cloud Providers: minikube, PKS, Azure, AWS-EKS, Openshift
+
+#Required parameter to be passed for all cloud providers
+cloudProvider: minikube
```

Comment:
Let's revisit this file as discussed. We don't want to repeat all the service yaml schema all over again. Let this is a simple key-value list with clear property names.